airflow/ui/src/queries/useLogs.tsx

```diff
@@ -49,8 +50,47 @@ const parseLogs = ({ data }: ParseLogsProps) => {
     return { data, warning };
   }
 
-  // eslint-disable-next-line react/no-array-index-key
-  const parsedLines = lines.map((line, index) => <p key={index}>{line}</p>);
+  let startGroup = false;
+  let groupLines: Array<string> = [];
+  let groupName = "";
+
+  // TODO: Add support for nested groups
+  /* eslint-disable react/no-array-index-key */
+  const parsedLines = lines.map((line, index) => {
+    if (line.includes("::group::") && !startGroup) {
+      startGroup = true;
+      groupName = line.split("::group::")[1] as string;
+    } else if (line.includes("::endgroup::")) {
+      startGroup = false;
+      groupLines.push(line);
+      const group = (
+        <details key={groupName}>
+          <summary data-testid={`summary-${groupName}`}>
+            <Text as="span" color="fg.info" cursor="pointer">
+              {groupName}
+            </Text>
+          </summary>
+          {groupLines.map((text, groupIndex) => (
+            <Text key={groupIndex} py={1}>
```

Comment:
This background adds some highlighting, `bg.muted` is the same that the background already has:
```suggestion
            <Text backgroundColor="bg.emphasized" key={groupIndex} py={1}>
```
It would then come out like:
![image](https://github.com/user-attachments/assets/c78cfb18-ab68-4d99-950b-a7948a0f28a3)

Reply:
I am not sure about the accessibility aspects of a lighter background for the text. I will leave this to @bbovenzi and would be happy to add if it's accessible.

Reply:
A lighter background is fine for a11y

Reply:
I think bg.emphasized or subtle is fine